src/cascadia/PublicTerminalCore/HwndTerminal.cpp

```diff
@@ -725,6 +724,10 @@ try
     {
         modifiers |= ControlKeyStates::EnhancedKey;
     }
+    if (vkey && keyDown && _uiaProviderInitialized)

```

Comment:
Considering doing a quick replacement of `_uiaProviderInitialized` --> `_uiaProvider` null check. Should accomplish the same thing.